src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Resources/EmailConfirmation.cshtml

```diff
@@ -1,31 +1,66 @@
+﻿@using TodoTemplate.Api.Models.Emailing
 
 @model EmailConfirmationModel
 
 <!DOCTYPE html>
-<html lang="en">
+<html>
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Confirm email</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="Content-TypeEntity" content="text/html; charset=UTF-8" />
+    <title>Email Confirmation</title>
 </head>
-<body>
-    Hello @Model.DisplayName
-    <br />
-    <br />
-    Thanks for using this app.
-    <br />
-    <br />
-    Please verify your email address.
-    <hr />
-    <a href="@Model.ConfirmationLink">Click to confirm!</a>
-    <br />
-    <br />
-    Or, copy the link below to your browser address bar:
-    <br />
-    <br />
-    @Model.ConfirmationLink
-    <hr />
+<body style="width:100%; height:100%; margin:0; padding:0; background-color:#FCFCFC; font-family:Arial,sans-serif;">
+    <main style="padding: 6% 4%;">
+        <table class="content-table" border="0" cellpadding="0" cellspacing="0" width="100%" height="100%"
+               style="border:1px solid #EDEBE9; border-radius:2px; padding:50px; text-align:center;">
+            <tr>
+                <td style="color:#0D2960;font-weight:600; font-size:24px; line-height:150%; padding-bottom:20px;">
+                    Welcome to Bit!
+                </td>
+            </tr>
+            <tr>
+                <td style="font-weight:400; font-size:18px; line-height:155%; padding-bottom:50px;">
+                    You're receiving this message because recently you have signed up for a BitProjectTemplate account.
+                    Confirm your email address by clicking the button below.
+                </td>
+            </tr>
+            <tr>
+                <td align="center">
+                    <table border="0" cellpadding="0" cellspacing="0">
+                        <tr>
+                            <td style="background-color:#0078D4;
+                                        border-radius:2px;
+                                        padding:10px 20px;">
+                                <a href="@Model.ConfirmationLink" style="text-decoration:none;
+                                color:#FFFFFF;
+                                font-weight:600;">
+                                    Confirm email
+                                </a>
+                            </td>
+                        </tr>
+                    </table>
+                </td>
+            </tr>
+        </table>
+        <table border="0" cellpadding="0" cellspacing="0" width="100%" height="100%"
+               style="padding:20px; text-align:center;">
+            <tr>
+                <td style="padding-bottom: 10px;">
+                    <img aalt="Company Logo" src="@($"{Model.HostUri}Resources/icon.png")" width="30" height="20" />
```

Comment:
Plz, change src like `@(new Uri(Model.HostUri, "Resources/icon.png").ToString())`

Reply:
Why use uri when it is finally going to become a string?
Also, the uri class receives UriKind in its second parameter. So if we use uri class, the code should be as follows :
`@(new Uri($"{Model.HostUri!}Resources/icon.png" , UriKind.Absolute))`
why this? @Cyrus-Sushiant 
when it is : `$"{Model.HostUri!}Resources/icon.png"`

Reply:
> Why use uri when it is finally going to become a string? Also, the uri class receives UriKind in its second parameter. So if we use uri class, the code should be as follows : `@(new Uri($"{Model.HostUri!}Resources/icon.png" , UriKind.Absolute))` why this? @Cyrus-Sushiant when it is : `$"{Model.HostUri!}Resources/icon.png"`

Because if the beginning and the end of the two addresses are "/", the time of concatenating two strings the final value is not correct.

> I also checked that if we use uri in cshtml file we might have trouble generating it. @Cyrus-Sushiant

When using Uri in the razor page you should use the System namespace in the first of the cshtml file